firmware/quadruna/BMS/src/app/states/app_chargeState.c

```diff
@@ -26,12 +54,12 @@ static void chargeStateRunOnTick100Hz(void)
     {
         const bool external_shutdown_occurred = !io_airs_isNegativeClosed();
         const bool charging_enabled           = app_canRx_Debug_StartCharging_get();
-        const bool is_charger_connected       = io_charger_isConnected();
+        const bool is_charger_connected       = app_canRx_BRUSA_IsConnected_get();
 
         bool has_charger_faulted = false;
         if (globals->ignore_charger_fault_counter >= CYCLES_TO_IGNORE_CHGR_FAULT)
         {
-            has_charger_faulted = io_charger_hasFaulted();
+            has_charger_faulted = app_canRx_BRUSA_Error_get();
```

Comment:
Perhaps we should also find a way to detect if the brusa has dropped off the bus? Technically the brusa should disable itself, but always safe to have measures on both ends